practice-two/src/components/CartItem.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+// Components
+import Button from './common/Button';
+import Typography from './common/Typography';
+
+// Remove Button Component
+const RemoveButton = ({ onClick = () => {} }) => (
+  <Button
+    aria-label="Remove cart item"
+    className="ease flex h-[17px] w-[17px] items-center justify-center rounded-full bg-neutral-500 transition-all duration-200"
+    onClick={onClick}
+  >
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth="1.5"
+      stroke="currentColor"
+      aria-hidden="true"
+      className="hover:text-accent-3 mx-[1px] h-4 w-4 text-white dark:text-black"
+    >
+      <path strokeLinecap="round" strokeLinejoin="round" d="M6 18L18 6M6 6l12 12" />
+    </svg>
+  </Button>
+);
+
+// Quantity Button Component
+const QuantityButton = ({ label = '', onClick = () => {}, children }) => (
+  <Button // github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.mdon
+    aria-label={label}
+    className="ease flex h-full min-w-[36px] max-w-[36px] flex-none items-center justify-center rounded-full px-2 transition-all duration-200 hover:border-neutral-800 hover:opacity-80"
+    onClick={onClick}
+  >
+    {children}
+  </Button>
+);
+
+// Product Image Component
+const ProductImage = ({ src, alt }) => (
+  <div className="relative h-16 w-16 cursor-pointer overflow-hidden rounded-md border border-neutral-300 bg-neutral-300 dark:border-neutral-700 dark:bg-neutral-900 dark:hover:bg-neutral-800">
+    <img src={src} alt={alt} width="64" height="64" className="h-full w-full object-cover" />
+  </div>
+);
+
+// Custom Typography Component
+const CustomTypography = ({
```

Comment:
Do we need this?

Reply:
This is not necessary, Removed this component